source/MRMesh/MRMatrix4.h

```diff
@@ -159,6 +164,37 @@ template <typename T>
 inline Matrix4<T> operator /( Matrix4<T> b, T a )
     { b /= a; return b; }
 
+template <typename T>
+constexpr Matrix3<T> Matrix4<T>::submatrix( int i, int j ) const noexcept
+{
+    Matrix3<T> res;
+    auto* resM = (T*) &res.x;
```

Comment:
will this function really work in a constant expression?
could you try to write for a test:
```
constexpr Matrix3 m3 = Matrix4().sumatrix(0,0);

Reply:
It does not work as constexpr, fixed.